app/models/customer.rb

```diff
@@ -9,4 +9,8 @@ def self.random
       id = Customer.pluck(:id).sample
       Customer.find(id)
     end
+
+    def transactions
```

Comment:
when you write instance methods on a class, be sure you test them - and by test them i mean test them on the unit level to ensure they're working properly 👍